### PR TITLE
toml.6.0.0 and toml_cconv.6.0.0

### PR DESCRIPTION
**Pull request by zapashcanon**

### CODE REVIEW

File: packages/toml/toml.6.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+authors: [ "Julien Sagot" "Emmanuel Surleau" "mackwic" "Andrew Rudenko" "orbifx" "c-cube"]
+maintainer: "OCamlPro <contact@ocamlpro.com>"
+homepage: "https://ocaml-toml.github.io/To.ml/"
+synopsis: "TOML parser"
+description: "The toml library provides a parser and serializer for Tom's Obvious Minimal Language v0.4.0, a minimal configuration file format. Helpful getters to retrieve data as OCaml primitive types are also supplied"
+build: [ "dune" "build" "-p" name]
+license: "LGPL3"
+depends: [
+  "ocaml" {>= "4.02"}
+  "dune"
```
Comment (mseri) [inline packages/toml/toml.6.0.0/opam:11]:
```suggestion
  "dune" { >="1.2"}
```

**Comment by zapashcanon** [comment]
@mseri, is this OK for you ? The failures are because of other packages missing lower bounds on OCaml or unrelated errors.

**Comment by kit-ty-kate** [comment]
Thanks a lot! Sorry for the delay. Could you return the fixes upstream. The syntax clean is just so we don't have to deal with oneline descriptions (it's easier to deal with when we edit files) and passing `-j %{jobs}%` to dune is rather important as it allows users to controle how much resources they want opam to take on their system. If it wasn't there dune assume "take all the cores".

Could you return this in the opam files upstream?

**Comment by zapashcanon** [comment]
Thanks ! I fixed the files in the upstream repository.

**Pull request merged**